tests/conftest.py

```diff
@@ -233,7 +233,7 @@ def _trigger_experiment_loader():
 def fixture_check_ping_for_experiment(trigger_experiment_loader):
     def _check_ping_for_experiment(experiment=None):
         control = True
-        timeout = time.time() + 60 * 2
+        timeout = time.time() + 60
```

Comment:
Ah, so this is a max 60 second loop, but with a 5 second pause (L251 below) between iterations.